packages/playground/src/components/profile_manager/ConnectWallet.vue

```diff
@@ -0,0 +1,374 @@
+                  #="{ props: validationProps }"
+                  ref="mnemonicInput"
+                >
+                  <div v-bind="tooltipProps">
+                    <VTextField
+                      readonly
+                      @focus="(event: Event) => (event.target as HTMLInputElement)?.removeAttribute('readonly')"
+                      @blur="(event: Event) => (event.target as HTMLInputElement)?.setAttribute('readonly', 'readonly')"
+                      id="mnemonic-text-field"
+                      :append-icon="enableReload && mnemonic !== '' ? 'mdi-reload' : ''"
+                      label="Mnemonic or Hex Seed"
+                      placeholder="Please insert your Mnemonic or Hex Seed"
+                      v-model="mnemonic"
+                      v-bind="{
+                        ...passwordInputProps,
+                        ...validationProps,
+                      }"
+                      autocomplete="off"
+                      :disabled="creatingAccount || connecting"
+                      @click:append="reloadValidation"
+                      ref="mnemonicRef"
+                    >
+                      <template v-slot:prepend-inner v-if="validationProps.hint || validationProps.error">
+                        <v-icon id="mnemonic-validation-icon" :color="validationProps.error ? 'red' : 'green'">
+                          {{ validationProps.error ? "mdi-close" : "mdi-check" }}
+                        </v-icon>
+                      </template>
+                    </VTextField>
+                  </div>
+                </InputValidator>
+              </PasswordInputWrapper>
+            </template>
+          </VTooltip>
+        </v-col>
+        <v-col cols="12" md="3">
+          <v-tooltip
+            id="keypair-tooltip"
+            location="top"
+            text="Using different keypair types will lead to a completely different account."
+          >
+            <template #activator="{ props }">
+              <v-select
+                id="keypair-select"
+                label="Keypair Type"
+                v-bind="props"
+                :items="[...keyType]"
+                item-title="name"
+                v-model="keypairType"
+                @update:modelValue="mnemonic !== '' ? reloadValidation() : null"
+              />
+            </template>
+          </v-tooltip>
+        </v-col>
+      </v-row>
+
+      <!-- create Account -->
+      <div id="create-account-container" class="d-flex flex-column flex-md-row justify-end mb-5">
+        <VBtn
+          id="create-account-btn"
+          class="mt-2 ml-sm-0 ml-md-3"
+          color="secondary"
+          variant="outlined"
+          :disabled="isValidForm || !!mnemonic || keypairType === KeypairType.ed25519 || isNonActiveMnemonic"
+          :loading="creatingAccount"
+          @click="openAcceptTerms = true"
+        >
+          create account
+        </VBtn>
+      </div>
+
```

Comment:
Button shouldn't be disabled if mnemonic is empty. In this case, I enterred my mn and chose ed25519 as a keypair type then, I removed my mnemonics and switched back to sr25519 but the button is still disabled.

![image](https://github.com/user-attachments/assets/2715358f-d6e6-4293-afe6-216d2df0057f)